qiita_db/metadata_template.py

```diff
@@ -1051,6 +1085,27 @@ def get_filepaths(self, conn_handler=None):
 
         return [(fpid, base_fp(fp)) for fpid, fp in filepath_ids]
 
+    def categories(self):
```

Comment:
Syntactic sugar... its easy to forget that there is actual leg work going on when things are properties. The nice thing about keeping it a function is it encourages the developer to acknowledge that there will be overhead. We could get around this through memoization, but we have not started down that route and there are additional consistency challenges there if we pursue that.